app/models/event.rb

```diff
@@ -55,6 +55,10 @@ def latest_provider_update
 
     private
 
+    def notify(event)
```

Comment:
I don't think this needs an argument, does it? You could just call `NotifyEventAppStore.perform_later(event: self)`, make the method signature much more straightforward.

Reply:
class method.. :)

Reply:
converted to an instance method